appveyor.yml

```diff
@@ -0,0 +1,30 @@
+build: false
+shallow_clone: true
+
+environment:
+
+    global:
+
+        distutils_use_sdk: 1
+        ets_toolkit: "qt4"
+
+    matrix:
+
+        - python: "C:/Python27-x64"
+          sdkver: "v7.0"
+
+cache:
+    - c:\temp
+
+init:
+    - ps: $Env:sdkbin = "C:\Program Files\Microsoft SDKs\Windows" + $Env:sdkver + "\Bin"
+    - ps: $ENV:sdkverpath = "C:\Program Files\Microsoft SDKs\Windows" + $Env:sdkver + "\Setup\WindowsSdkVer.exe"
+    - ps: $ENV:path = $Env:python + ";" + $env:python + "\scripts;" + $Env:sdkbin + ";" + $Env:path
+
+install:
+    - ps: if ((Test-Path "C:\temp") -eq 0)  {mkdir C:\temp}
+    - ps: python -m pip install --upgrade --no-binary wheel pip
+    - ps: pip install --upgrade wheel
```

Comment:
I think that we do not need to do this since you already upgrade wheel in the previous line